modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,61 @@
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
```

Comment:
You need to have a comment indicating which assembly the file is included in, giving it it's relative path below the openshift-docs directory. For example: 

// Module included in the following assemblies:
//
// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc